app/pods/cases/case/subcases/overview/template.hbs

```diff
@@ -1,18 +1,20 @@
 {{page-title (t "subcases")}}
-{{subcases/subcases-overview-header
-  model=model
-  isShowingProcess=isShowingProcess
-  backLink=(action "backLinkAction")
-  refresh=(action "refresh")
-}}
-<div class="auk-scroll-wrapper__body">
-  {{#if (gt (await model.subcases.length) 0)}}
-
-    <Subcases::SubcasesProcess @subcases={{await model.subcases}}/>
 
+<Subcases::SubcasesOverviewHeader
+        @case={{this.model.case}}
+        @isShowingProcess={{this.isShowingProcess}}
+        @backLink={{this.backLinkAction}}
+        @onSaveSubcase={{perform this.saveSubcase}}
+        @onSaveCase={{perform this.saveCase}}
+/>
+<div class="auk-scroll-wrapper__body">
+  {{#if (gt (await this.model.subcases.length) 0)}}
```

Comment:
Dit kan korter geschreven worden door gewoon `this.model.subcases`. Dit gaat als falsy aanzien worden indien het een lege array is. 

De `await` is niet expliciet nodig omdat de Ember template automatisch update als de relation promise resolved.